### ISSUE TITLE
The scanner does not respect the language of the phone

### ISSUE THREAD

**Issue by teolemon** [openfoodfacts/openfoodfacts-androidapp#1981]
The scanner does not respect the language of the phone.
If a product is available in English and Spanish, with English as the main language, but Spanish as the language of the app, the product will be displayed in English.
The app should display the product in Spanish.

**Comment by jaindiv26** [comment]
@teolemon Yes, the scanner does not respect the language of the phone, because it's depends on the app language. Whatever the language of the phone is, the app will always follow it's own language. But what we can do is, at the time of first time opening of the app, we run this line "Locale.getDefault().getDisplayLanguage();" to get the device language and set it as default of the app language.

For example when the app is first run in the phone and the phone language is Espanol we get the device language and set the same in the app. Afterwards whatever the user changes the language of the app, it's his/her choice.

And the product is displayed according to the app language. See screenshots.
![screenshot_2018-12-29-22-05-38-911_openfoodfacts github scrachx openfood](https://user-images.githubusercontent.com/32436867/50540342-2e48ff00-0bb6-11e9-87aa-03d211a7ce73.png)
![screenshot_2018-12-29-22-06-26-625_openfoodfacts github scrachx openfood](https://user-images.githubusercontent.com/32436867/50540344-2ee19580-0bb6-11e9-9129-2cfa370b2cac.png)
![screenshot_2018-12-29-22-06-50-842_openfoodfacts github scrachx openfood](https://user-images.githubusercontent.com/32436867/50540345-2ee19580-0bb6-11e9-948a-fefec390e446.png)
